doc/user/content/ingest-data/cdc-sql-server.md

```diff
@@ -0,0 +1,172 @@
+---
+title: "SQL Server CDC using Kafka and Debezium"
+description: "How to propagate Change Data Capture (CDC) data from a SQL Server database to Materialize"
+aliases:
+  - /guides/cdc-sql-server/
+  - /integrations/cdc-sql-server/
+  - /connect-sources/cdc-sql-server/
+menu:
+  main:
+    parent: "sql-server"
+    name: "Using Kafka and Debezium"
+    identifier: "sql-server-dbz"
+    weight: 15
+---
+
+Change Data Capture (CDC) allows you to track and propagate changes in a SQL Server database to downstream consumers. In this guide, we’ll cover how to use Materialize to create and efficiently maintain real-time materialized views on top of CDC data.
+
+## Kafka + Debezium
+
+Utilize [Debezium](https://debezium.io/) and the [Kafka source](/sql/create-source/kafka/#using-debezium) to propagate CDC data from SQL Server to Materialize. Debezium captures row-level changes resulting from `INSERT`, `UPDATE`, and `DELETE` operations in the upstream database and publishes them as events to Kafka using Kafka Connect-compatible connectors.
+
+### Database Setup
+
+Before deploying a Debezium connector, ensure that the upstream database is configured to support CDC.
+
+1.  Enable CDC on the SQL Server instance and database:
+
+    ```sql
+    -- Enable CDC on the SQL Server instance
+    USE MyDB
+    GO
+    EXEC sys.sp_cdc_enable_db
+    GO
+    ```
+
+### Deploy Debezium
+
+**Minimum requirements:** Debezium 1.5+
+
+Debezium is deployed as a set of Kafka Connect-compatible connectors. First, define a SQL Server connector configuration, then start the connector by adding it to Kafka Connect.
+
+{{< tabs >}}
+{{< tab "Debezium 1.5+">}}
+
+1.  Create a connector configuration file and save it as `register-sqlserver.json`:
```

Comment:
What about using the link to [the configuration example](https://debezium.io/documentation/reference/1.9/connectors/sqlserver.html#sqlserver-example-configuration) that includes also additional details rather than the json?

Reply:
That example in the Debezium docs is missing some crucial configuration settings like the Avro Converter and the CSR details so we can't use it directly.